src/app/common/directives/circular-tessellation.js

```diff
@@ -98,6 +98,7 @@ function circularTessellationDirective($location) {
 			}
 
 			function redraw() {
+				scope.circularTessellation.cancel();
```

Comment:
Do you think it would be cleaner if the below `paper.clear()` call was then moved within the `cancel()` method?

i.e. remove the `paper.clear()` call below and add `this.paper.clear() within the `cancel()` method?